matching/models.py

```diff
@@ -52,3 +55,14 @@ class UserTripMatches(models.Model):
     match_status = models.CharField(
         max_length=20, choices=MatchStatus, default=MatchStatusEnum.PENDING.value
     )
+
+    def save(self, *args, **kwargs):
+        logger = logging.getLogger()
+        logger.info("In the save function Match status:")
+        logger.info(self.match_status)
+        if self.match_status == MatchStatusEnum.MATCHED.value:
+            t = Thread.objects.get_or_create(
```

Comment:
I believe get_or_create returns a tuple so we may want to do the variable declaration as t, _